src/www/build_cordova_action.sh

```diff
@@ -14,7 +14,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Builds the TypeScript in this folder, placing a web app at /www (for Cordova).
-
-tsc -p src/www
-rsync -ac --exclude '*.ts' src/www/ www/
+yarn install --check-files
```

Comment:
Can we move this to webpack (common/base) and document why we need to run it? [webpack-shell-plugin-next](https://www.npmjs.com/package/webpack-shell-plugin-next) may be useful.

Reply:
Nice find!